query/src/main/java/io/keen/client/java/QueryType.java

```diff
@@ -0,0 +1,65 @@
+package io.keen.client.java;
+
+/**
+ * QueryType specifies which query to run.
+ *
+ * Created by claireyoung on 6/16/15.
+ */
+public enum QueryType {
+    COUNT_RESOURCE,
+    COUNT_UNIQUE,
+    MINIMUM_RESOURCE,
+    MAXIMUM_RESOURCE,
+    AVERAGE_RESOURCE,
+    MEDIAN_RESOURCE,
+    PERCENTILE_RESOURCE,
+    SUM_RESOURCE,
+    SELECT_UNIQUE_RESOURCE;
+
+    /**
+     * Gets the query type to send in the query HTTP request.
+     *
+     * @param type the QueryType
+     * @return  the query string to specify in the HTTP request.
+     *
+     */
+    public static final String getQueryType (QueryType type) {
```

Comment:
Actually, I looked here: http://stackoverflow.com/questions/3978654/best-way-to-create-enum-of-strings - it looks like this would get rid of the ugly switch() statement.